### PR TITLE
Access Control Quick-Start page updates

### PR DESCRIPTION
**Pull request by eanders**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

This fixes some typos, gives popovers a close button, and only adds (Legacy) to the menu if there is at least one user using the new version of access control.

## Type of change
[//]: # 'remove options that are not relevant'
- [x] Bug fix
- [x] New feature (adds functionality)

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: app/views/admin/access_overviews/index.haml [hunk 1]
```diff
@@ -14,24 +14,25 @@
         %p.access-description
           Each
           %span.access-control.color-tag<
-            = link_to 'access control', '#', tabindex: '0', data: {container: '.access-overview', toggle: :popover, placement: :top, html: true, trigger: :focus, content: "<p>Accss Controls are used to grant users access to data and permissions.  Each access control consists of one Role, one User Group, and one Collection. Access is governed by the intersection of permission from the role, entities from the collection, and users from the user group.</p><p>#{link_to 'Manage Access Controls', admin_access_controls_path}</p>", title: 'Access Control'}
```
Comment (gigxz) [inline app/views/admin/access_overviews/index.haml:17]: Typo a few lines above: `goverend` => `governed`

Comment (gigxz) [reply app/views/admin/access_overviews/index.haml:17]: Also suggest adding a comma:
`are used to grant access to data*,* and `

**Review by gigxz — APPROVED** [review]
thanks!

File: app/views/admin/access_overviews/index.haml (hunk 1 above)
Comment (eanders) [reply app/views/admin/access_overviews/index.haml:17]: That's what I get for copying out of figma I guess.

**Pull request merged**